nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/PlanQK-Platform_w1-wip1/README.md

```diff
@@ -0,0 +1,27 @@
+# PlanQK Platform [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)  
+
+> This node type provides the API key to access the PlanQK Platform
+
+## Properties
+
+- `PlanQKApiKey`: personal access token for the PlanQK platform. Should be set to `get_input: PlanqkApiKey`.
```

Comment:
Set `get_input: PlanqkApiKey` as the default value in the properties definition.